CoseSignTool.Tests/ValidateCommandTests.cs

```diff
@@ -155,39 +155,73 @@ public void ValidateUntrustedFails()
     [TestMethod]
     public void ValidateIndirectSucceedsWithRootPassedIn()
     {
-        string payloadFile = Utils.GetPayloadFile();
+        string payloadFile = FileSystemUtils.GeneratePayloadFile();
 
         // sign indirectly
         var msgFac = new IndirectSignatureFactory();
         byte[] signedBytes = msgFac.CreateIndirectSignatureBytes(
-        payload: File.ReadAllBytes(payloadFile),
+            payload: File.ReadAllBytes(payloadFile),
             contentType: "application/spdx+json",
-            signingKeyProvider: new X509Certificate2CoseSigningKeyProvider(SelfSignedCert)).ToArray();
+            signingKeyProvider: new X509Certificate2CoseSigningKeyProvider(SelfSignedCert))
+            .ToArray();
 
         using FileStream coseFile = new(payloadFile + ".cose", FileMode.Create);
         coseFile.Write(signedBytes);
         coseFile.Seek(0, SeekOrigin.Begin);
 
         // setup validator
         var validator = new ValidateCommand();
-        var result = validator.RunCoseHandlerCommand(coseFile,
-                                                     new FileInfo(payloadFile),
-                                                     [SelfSignedCert],
-                                                     X509RevocationMode.Online,
-                                                     null,
-                                                     false);

```

Comment:
I may be confused, but I thought these were required params for `RunCoseHandlerCommand()`?

Reply:
Nope. They have default values.